JetNews/app/src/main/java/com/example/jetnews/ui/SwipeToRefresh.kt

```diff
@@ -83,7 +86,7 @@ fun SwipeToRefreshLayout(
         //  workaround for a bug in the SwipableState API. Currently, state.value is a duplicated
         //  source of truth of refreshingState.
         DisposableEffect(refreshingState) {
-            state.animateTo(refreshingState)
+            scope.launch { state.animateTo(refreshingState) }
```

Comment:
If you're calling coroutines here, use `LaunchedEffect(refreshingState)` instead, you don't need the scope here